# HG changeset patch
# User Patrick Noffke <patrick.noffke@gmail.com>
# Date 1376339528 18000
#      Mon Aug 12 15:32:08 2013 -0500
# Node ID a594e0d980eb10b1ca43ff99cd70ec3a90554f9e
# Parent  ea9992fd9c89130a9fb379833d13bc8d7e491934
Expand all macros for doxygen documentation.

diff --git a/doc/doxyhtml/Doxyfile.in b/doc/doxyhtml/Doxyfile.in
--- a/doc/doxyhtml/Doxyfile.in
+++ b/doc/doxyhtml/Doxyfile.in
@@ -179,23 +179,20 @@ SORT_MEMBERS_CTORS_1ST = YES
 #---------------------------------------------------------------------------
 # Configuration options related to the preprocessor
 #---------------------------------------------------------------------------
 
 # On by default, but let's be explicit
 
 ENABLE_PREPROCESSING   = YES
 
-# Expand the DEFUN family of macros
+# Expand all macros
 
 MACRO_EXPANSION = YES
-EXPAND_ONLY_PREDEF = YES
-EXPAND_AS_DEFINED = DEFUN DEFUN_DLD  # As defined in the Octave source
-                                     # code, i.e. not overriden by this
-                                     # config file
+EXPAND_ONLY_PREDEF = NO
 
 # So that features that are behind #ifdef HAVE_FOO macros get processed by Doxygen
 
 PREDEFINED = HAVE_ARPACK=1          \
              HAVE_CHOLMOD=1         \
              HAVE_CCOLAMD=1         \
              HAVE_CURL=1            \
              HAVE_CXSPARSE=1        \
